packages/analytics/analytics-chart/src/composables/useLineChartOptions.ts

```diff
@@ -60,7 +60,7 @@ export default function useLinechartOptions(chartOptions: LineChartOptions) {
 
   Tooltip.positioners.lineChartTooltipPosition = function(elements, position) {
     // Happens when nothing is found
-    if (!elements.length) {
+    if (!elements.length || chartOptions.tooltipState.locked) {
```

Comment:
Shouldn't we do conditionally check for `tooltipState?.locked` throughout?  Since `locked` is defined as optional in chart-options.ts...

Reply:
Why? If you don't use it, it'll just be undefined in which case the locked check here will essentially behave like it's not locked, as expected.

Reply:
TooltipState?.locked 

This conditional is for the case that tooltipState might be undefined

Reply:
not necessary